integration/kubernetes/helm-chart/alluxio/templates/logserver/deployment.yaml

```diff
@@ -0,0 +1,106 @@
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+
+{{ if .Values.logserver.enabled -}}
+{{- $name := include "alluxio.name" . }}
+{{- $fullName := include "alluxio.fullname" . }}
+{{- $chart := include "alluxio.chart" . }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ $fullName }}-logserver
+  labels:
+    name: {{ $fullName }}-logserver
+    app: {{ $name }}
+    chart: {{ $chart }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    role: alluxio-logserver
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ $name }}
+      role: alluxio-logserver
+      name: {{ $fullName }}-logserver
+  template:
+    metadata:
+      labels:
+        name: {{ $fullName }}-logserver
+        app: {{ $name }}
+        chart: {{ $chart }}
+        release: {{ .Release.Name }}
+        heritage: {{ .Release.Service }}
+        role: alluxio-logserver
+    spec:
+      containers:
+        - name: alluxio-logserver
+          image: {{ .Values.image }}:{{ .Values.imageTag }}
+          imagePullPolicy: {{ .Values.imagePullPolicy }}
+          securityContext:
+            runAsUser: {{ .Values.user }}
+            runAsGroup: {{ .Values.group }}
+          {{- if .Values.logserver.resources  }}
+{{ include "alluxio.logserver.resources" . | indent 10 }}
+          {{- end }}
+          command: ["tini", "--", "/entrypoint.sh"]
+            {{- if .Values.logserver.args }}
+          args:
+{{ toYaml .Values.logserver.args | trim | indent 12 }}
+            {{- end }}
+          env:
+            {{- range $key, $value := .Values.logserver.env }}
+          - name: "{{ $key }}"
+            value: "{{ $value }}"
+            {{- end }}
+          envFrom:
+          - configMapRef:
+              name: {{ $fullName }}-config
+          ports:
+          - containerPort: {{ .Values.logserver.ports.logging }}
+            name: logging
+          volumeMounts:
+            {{- if not (eq .Values.logserver.volumeType "none") }}
+          - name: alluxio-logs
+            mountPath: /opt/alluxio/logs
+            {{- end }}
+            {{- if .Values.secrets }}
+              {{- if .Values.secrets.logserver }}
+            {{- include "alluxio.master.secretVolumeMounts" . }}
```

Comment:
why does the logserver need master secrets?

Reply:
Oops, should be logserver secrets